include/picongpu/fields/incidentField/profiles/ExpRampWithPrepulse.hpp

```diff
@@ -244,8 +241,9 @@ namespace picongpu::fields::incidentField
                     //        "and physically very unplausible, check the params for misunderstandings!");
                     //}
 
-                    env += (1._X - ramp_when_peakpulse) * gauss(runTime - Unitless::endUpramp);
-                    env += AMP_PREPULSE * gauss(runTime - Unitless::TIME_PREPULSE);
+                    env += (1._X - ramp_when_peakpulse) * gauss(runTime - Unitless::endUpramp,
+                                                                Unitless::PULSE_DURATION);

```

Comment:
Here we need a line break according to code style. 
```suggestion
                    env += (1._X - ramp_when_peakpulse)
                        * gauss(runTime - Unitless::endUpramp, Unitless::PULSE_DURATION);
```